### ISSUE TITLE
_TIME_BITS=64 is allowed only with _FILE_OFFSET_BITS=64

### ISSUE THREAD

**Issue by introspectionism** [michaelrsweet/lprint#65]
```
In file included from /usr/include/features.h:392,
                 from /usr/include/sys/types.h:25,
                 from /usr/include/cups/cups.h:19,
                 from /usr/include/pappl/base.h:12,
                 from /usr/include/pappl/device.h:12,
                 from /usr/include/pappl/pappl.h:12,
                 from lprint.h:19,
                 from lprint-testpage.c:14:
/usr/include/features-time64.h:26:5: error: #error "_TIME_BITS=64 is allowed only with _FILE_OFFSET_BITS=64"
   26 | #   error "_TIME_BITS=64 is allowed only with _FILE_OFFSET_BITS=64"
```

System:
```
OS: Manjaro Linux x86_64 
Kernel: 5.17.14-1-MANJARO 
```

**Comment by michaelrsweet** [comment]
Sigh… I hate glibc’s inconsistencies…

**Comment by michaelrsweet** [comment]
[master c7b72dd] Fix glibc issue (Issue #65), bump version to 1.2.0, update PAPPL version in snap.
